coding-guide.html

```diff
@@ -102,6 +102,47 @@
 		<li>We should attempt to maintain API compatibility when possible, though at this point in the project's lifecycle it is more important to make things good rather than avoid breakage.</li>
 		</ul>
 
+		<h2>Streams API</h2>
+		<p>Kafka's Streams API (aka Kafka Streams) uses a few more additional coding guidelines.
+                   All contributors should follow those the get a high quality and uniform code base.
+                   Some rules help to simplify PR reviews and thus make the live of all contributors easier.</p>
+		<ul>
+			<li>Use <code>final</code> when possible.
+                            This holds for all class members, local variables, loop variables, and method parameters.</li>
+			<li>Write modular and thus testable code. Refactor if necessary!</li>
+			<li>Avoid large PRs (recommended is not more the 500 lines per PR).
+                            Many JIRAs requires larger code changes; thus, split the work in multiple PRs and create according sub-task on the JIRA to track the work.</li>
+			<li>All public APIs must have JavaDocs.</li>
+			<li>Verify if JavaDocs are still up to date or if they need to be updated.</li>
+			<li>JavaDocs: Write grammatically correct sentences and use punctuation marks correctly.</li>
+			<li>Use proper markup (e.g., <code>{@code null}</code>).</li>
+			<li>Update the documentation on the Kafka webpage (i.e., within folder <code>docs/</codes>.
+                            Doc changes are not additional work (i.e. no follow up PRs) but part of the actuall PR (can also be a sub-tasks).</li>
```

Comment:
@mjsax 

nit: actuall -> actual

Reply:
Two more nits:
> All contributors should follow those the get a high quality and uniform code base.

Nit: follow those the get -> follow those to get

>  thus make the live of all contributors easier

Nit: the live -> the life

Reply:
Thanks @Kamal15 @Sharon2017 !